### PR TITLE
feat: openapi response schema

### LINKED ISSUE #930: Response schema missing in openapi docs
Screenshot 2024-08-17 at 18 58 39

### PR DESCRIPTION
**Pull request by VishnuSanal**
## Description

This PR fixes #930

## Summary

This PR does add the openapi response schema

## PR Checklist

Please ensure that:

- [x] The PR contains a descriptive title
- [x] The PR contains a descriptive summary of the changes
- [x] You build and test your changes before submitting a PR.
- [ ] You have added relevant documentation
- [x] You have added relevant tests. We prefer integration tests wherever possible

## Pre-Commit Instructions:

- [x] Ensure that you have run the [pre-commit hooks](https://github.com/sparckles/robyn#%EF%B8%8F-to-develop-locally) on your PR.

### CODE REVIEW

File: robyn/types.py [hunk 1]
```diff
@@ -1,3 +1,4 @@
+import typing
 from dataclasses import dataclass
 from typing import Optional
```
Comment (sansyrox) [inline robyn/types.py:3]: import `TypedDict` here

**Comment by VishnuSanal** [comment]
- [ ] docs

**Comment by VishnuSanal** [comment]
@sansyrox updated the docs, PTAL.

File: docs_src/src/pages/documentation/example_app/openapi.mdx [hunk 2]
```diff
@@ -198,12 +200,19 @@ class CreateItemBody(TypedDict):
...
+class CreateResponse(JSONResponse):
+    success: bool
+    items_changed: int
+
+
 @app.post("/")
-def create_item(request, body=CreateItemBody):
-    return request.body
+def create_item(request: Request, body=CreateItemBody) -> CreateResponse:
+    return CreateResponse(success=True, items_changed=2)
```
Comment (sansyrox) [inline docs_src/src/pages/documentation/example_app/openapi.mdx:210]:
```suggestion
def create_item(request: Request, body=CreateItemBody) -> CreateResponse:
    return {"success": True, "items_changed": 2}
```

Comment (VishnuSanal) [reply docs_src/src/pages/documentation/example_app/openapi.mdx]: Whyy?!

**Review by sansyrox — APPROVED** [review]
LGTM! 👍 ❇️

**Pull request merged**